tests/utils/object_store/test_s3_object_store.py

```diff
@@ -21,17 +21,13 @@ def _worker(bucket: str, tmp_path: pathlib.Path, tid: int):
 # This test requires properly configured aws credentials; otherwise the s3 client would hit a NoCredentialsError
 # when constructing the Session, which occurs before the bug this test checks
 @pytest.mark.remote
-def test_s3_object_store_multi_threads(tmp_path: pathlib.Path):
+def test_s3_object_store_multi_threads(tmp_path: pathlib.Path, s3_bucket: str):
```

Comment:
Just a general question on `ObjectStoreLogger` tests, do we have any tests that vary `flush_interval`?  I noticed while working on a demo that setting this value to a small value can cause some hangs.  Probably a topic for another issue but just curious.

Reply:
I don't think we vary the flush interval.
Can you file a bug report for an example where it hangs?